### PR TITLE
fix: update CLI yaml to add session features and Auth Hooks

### PR DESCRIPTION
**Pull request by J0**
## I have read the [CONTRIBUTING.md](https://github.com/supabase/supabase/blob/master/CONTRIBUTING.md) file.

YES

## What kind of change does this PR introduce?

Update CLI YAML to include Auth Hooks and session control. Follow up to #30830

### CODE REVIEW

File: apps/docs/spec/cli_v1_config.yaml [hunk 1]
```diff
@@ -1142,6 +1142,39 @@ parameters:
       - name: 'Auth Server configuration'
         link: 'https://supabase.com/docs/reference/auth'
 
+  - id: 'auth.hook.<hook_name>.enabled'
+    title: 'auth.hook.<hook_name>.enabled'
+    tags: ['auth']
+    required: false
+    default: 'true'
+    description: |
+      Enable Auth Hook. Possible values for `hook_name` are: `custom_access_token`, `send_sms`, `send_email`, `mfa_verification_attempt`, `password_verification_attempt`.
```
Comment (J0) [inline apps/docs/spec/cli_v1_config.yaml:1151]:
```suggestion
      Enable Auth Hook. Possible values for `hook_name` are: `custom_access_token`, `send_sms`, `send_email`, `mfa_verification_attempt`, and `password_verification_attempt`.
```

File: apps/docs/spec/cli_v1_config.yaml [hunk 2]
```diff
@@ -1142,6 +1142,39 @@ parameters:
...
+      - name: 'Auth Hooks'
+        link: 'https://supabase.com/docs/guides/auth/auth-hooks'
+
+  - id: 'auth.hook.<hook_name>.secrets'
+    title: 'auth.hook.<hook_name>.secrets'
+    tags: ['auth']
+    required: false
+    default: ''
+    description: |
+      Configure when using a HTTP Hooks. Takes a list of base64 comma separated values to allow for secret rotation. Currently, Supabase Auth uses only the first value.
```
Comment (J0) [inline apps/docs/spec/cli_v1_config.yaml:1173]:
```suggestion
      Configure when using a HTTP Hooks. Takes a list of base64 comma separated values to allow for secret rotation. Currently, Supabase Auth uses only the first value in the list.
```

File: apps/docs/spec/cli_v1_config.yaml [hunk 3]
```diff
@@ -1142,6 +1142,39 @@ parameters:
...
+      - name: 'Auth Hooks'
+        link: 'https://supabase.com/docs/guides/auth/auth-hooks'
+
+  - id: 'auth.hook.<hook_name>.uri'
+    title: 'auth.hook.<hook_name>.uri'
+    tags: ['auth']
+    required: false
+    default: ''
+    description: |
+      URI of hook to invoke. Should be a HTTPS function or Postgres function taking the form: `pg-functions://<database>/<schema>/<function-name>`. For example, `pg-functions://postgres/auth/custom-access-token-hook`.
```
Comment (J0) [inline apps/docs/spec/cli_v1_config.yaml:1162]:
```suggestion
      URI of hook to invoke. Should be a http or https function or Postgres function taking the form: `pg-functions://<database>/<schema>/<function-name>`. For example, `pg-functions://postgres/auth/custom-access-token-hook`.
```

File: apps/docs/spec/cli_v1_config.yaml [hunk 4]
```diff
@@ -1254,6 +1287,28 @@ parameters:
       - name: 'Auth Multi-Factor Authentication'
         link: 'https://supabase.com/docs/guides/auth/auth-mfa'
 
+  - id: 'auth.sessions.timebox'
+    title: 'auth.sessions.timebox'
+    tags: ['auth']
+    required: true
```
Comment (J0) [inline apps/docs/spec/cli_v1_config.yaml:1293]:
```suggestion
    required: false
```

File: apps/docs/spec/cli_v1_config.yaml [hunk 5]
```diff
@@ -1254,6 +1287,28 @@ parameters:
...
+    description: |
+      Force log out after the specified duration. Sample values include: '50m', '20h'.
+    links:
+      - name: 'Auth Sessions'
+        link: 'https://supabase.com/docs/guides/auth/sessions'
+
+  - id: 'auth.sessions.inactivity_timeout'
+    title: 'auth.sessions.inactivity_timeout'
+    tags: ['auth']
+    required: true
```
Comment (J0) [inline apps/docs/spec/cli_v1_config.yaml:1304]:
```suggestion
    required: false
```

File: apps/docs/spec/cli_v1_config.yaml [hunk 6]
```diff
@@ -1142,6 +1142,39 @@ parameters:
       - name: 'Auth Server configuration'
         link: 'https://supabase.com/docs/reference/auth'
 
+  - id: 'auth.hook.<hook_name>.enabled'
+    title: 'auth.hook.<hook_name>.enabled'
+    tags: ['auth']
+    required: false
+    default: 'true'
```
Comment (J0) [inline apps/docs/spec/cli_v1_config.yaml:1149]:
```suggestion
    default: 'false'
```

File: apps/docs/spec/cli_v1_config.yaml [hunk 7]
```diff
@@ -1254,6 +1287,28 @@ parameters:
...
+      Force log out after the specified duration. Sample values include: '50m', '20h'.
+    links:
+      - name: 'Auth Sessions'
+        link: 'https://supabase.com/docs/guides/auth/sessions'
+
+  - id: 'auth.sessions.inactivity_timeout'
+    title: 'auth.sessions.inactivity_timeout'
+    tags: ['auth']
+    required: false
+    default: '8h'
```
Comment (hf) [inline apps/docs/spec/cli_v1_config.yaml:1305]: Is it really default 8h?

**Review by hf — APPROVED** [review]

File: apps/docs/spec/cli_v1_config.yaml (hunk 7 above)
Comment (J0) [reply apps/docs/spec/cli_v1_config.yaml:1305]: Good catch, it's not, 8h is the example from testdata. Updating now..

File: apps/docs/spec/cli_v1_config.yaml [hunk 8]
```diff
@@ -1254,6 +1287,28 @@ parameters:
       - name: 'Auth Multi-Factor Authentication'
         link: 'https://supabase.com/docs/guides/auth/auth-mfa'
 
+  - id: 'auth.sessions.timebox'
+    title: 'auth.sessions.timebox'
+    tags: ['auth']
+    required: false
+    default: '24h'
```
Comment (J0) [inline apps/docs/spec/cli_v1_config.yaml:1294]:
```suggestion
    default: ''
```

**Pull request merged**